docs/content/topics/project-developers/projects.md

```diff
@@ -79,206 +44,209 @@ spec:
+cases -- for instance, ones wherein a script is meant to directly modify
+Kubernetes resources in a cluster -- a user who has credentials for the
+underlying Kubernetes cluster may wish to learn what namespace a project is
+assigned to so they can directly modify resources such as Kubernetes
+`ServiceAccount`s or `RoleBinding`s in that namespace.
 
-[Secrets]: /topics/project-developers/secrets
-[Authorization]: /topics/administrators/authorization
+This information can be retrieved from the `kubernetes.namespace` section after
+using the `brig project get` command as described in the previous section.
 
-## Project namespaces
+## Project Secrets
 
-Brigade creates a unique namespace on the underlying substrate (Kubernetes)
-corresponding to each project. Although most users shouldn't have a need to
-inspect resources under a project namespace on the substrate, to see which
-unique namespace a project is assigned, run the `brig project get` command and
-note the `kubernetes.namespace` value.  For example:
+The scripts executed by a project's workers often need to make use of sensitive
+information that should not be hard-coded into the project definition or script.
+To manage such details, the CLI provides a suite of `brig project secret`
+commands.
 
-```plain
-$ brig project get --id hello-world --output yaml
+To set a secret:
 
-apiVersion: brigade.sh/v2
-description: The simplest possible example
-kind: Project
-kubernetes:
-  namespace: brigade-97cd352f-90e1-48d0-8797-4f7867a72bd3
-metadata:
-  created: "2021-08-11T17:47:07.555Z"
-  id: hello-world
-spec:
-  eventSubscriptions:
-  - source: brigade.sh/cli
-    types:
-    - exec
-  workerTemplate:
-    defaultConfigFiles:
-      brigade.js: |
-        console.log("Hello, World!");
-    logLevel: DEBUG
-    useWorkspace: false
+```shell
+$ brig project secret set --project <project id> --set <key>=<value>
 ```
 
-## Git-based projects
-
-### Using SSH Keys
+To set many secrets in bulk from a "flat" JSON or YAML file:
 
-You can use SSH keys and a `git+ssh` URL to secure a private repository.
-
-In this case, your project's `cloneURL` should be of the form
-`git@github.com:<org>/<repo>.git` and you will need to add the SSH
-_private key_ as a secret to the project with the key `gitSSHKey`.
-
-For example, if project secrets are contained in a `secrets.yaml` file, the
-private key would be added like so:
-
-```yaml
-gitSSHKey: |-
-  -----BEGIN RSA PRIVATE KEY-----
-  IIEpAIBAAKCAg1wyZD164xNLrANjRrcsbieLwHJ6fKD3LC19E...
-  ...
-  ...
-  -----END RSA PRIVATE KEY-----
+```shell
+$ brig project secret set --project <project id> --file secrets.yaml
 ```
 
-The project secrets can then be updated via the usual brig command:
+To list secrets for a project:
 
 ```shell
-$ brig project secrets set --file secrets.yaml
+$ brig project secret list --project <project id>
 ```
 
-### Using other Git providers
+The above command will display all keys, but will redacted values.
```

Comment:
```suggestion
The above command will display all keys, but with redacted values.
```

Reply:
I must have mixed up "with redacted" and "will redact" and landed on a combination of the two that made no sense!

Thanks for spotting this!